client/src/components/reactions/reactions.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import classNames from 'classnames/bind'
+// Styles
+import styles from './reactions.scss'
+
+const css = classNames.bind(styles)
+
+const mapStateToProps = (state) => {
+  const users = (state.persist || {}).users || {}
+  return { users }
+}
+
+class Reactions extends Component {
+  constructor(props, context) {
+    super(props, context)
```

Comment:
What is context needed for?

Reply:
Needs to be passed for React's sake I think, maybe Redux I think?

Reply:
Maybe not tbh